### PR TITLE
fix(build.sh): Quotes on variable causes build detect to break

### PR DESCRIPTION
**Pull request by n0n0x**
```
$ git push deis master
Enumerating objects: 964, done.
Counting objects: 100% (964/964), done.
Delta compression using up to 8 threads
Compressing objects: 100% (601/601), done.
Writing objects: 100% (964/964), 177.31 KiB | 10.43 MiB/s, done.
Total 964 (delta 576), reused 573 (delta 340), pack-reused 0
remote: Resolving deltas: 100% (576/576), done.
Starting build... but first, coffee!
...
...
-----> Restoring cache...
       No cache file found. If this is the first deploy, it will be created now.
/builder/build.sh: line 162: /tmp/buildpacks/*/bin/detect: No such file or directory
-----> Unable to select a buildpack
```

```
slug@78a0f0b805eb:~$ buildpacks=($buildpack_root/*)
slug@78a0f0b805eb:~$ for buildpack in "${buildpacks[@]}"; do   echo "- $buildpack"; done
- /tmp/buildpacks/01-multi
- /tmp/buildpacks/02-clojure
- /tmp/buildpacks/03-go
- /tmp/buildpacks/04-gradle
- /tmp/buildpacks/05-grails
- /tmp/buildpacks/06-java
- /tmp/buildpacks/07-nodejs
- /tmp/buildpacks/08-php
- /tmp/buildpacks/09-play
- /tmp/buildpacks/10-python
- /tmp/buildpacks/11-ruby
- /tmp/buildpacks/12-scala


slug@78a0f0b805eb:~$ buildpacks=("$buildpack_root/*")
slug@78a0f0b805eb:~$ for buildpack in "${buildpacks[@]}"; do   echo "- $buildpack"; done
- /tmp/buildpacks/*
slug@78a0f0b805eb:~$
```

### CODE REVIEW

**Review by Cryptophobia — CHANGES_REQUESTED** [review]
Looks like build broke because of quotes missing: https://travis-ci.org/github/teamhephy/slugbuilder/builds/748734755

Can you make it pass without quotes? Can you try like this:
```
buildpacks=(${buildpack_root}/*)
```

File: rootfs/builder/build.sh
Comment (Cryptophobia) [inline rootfs/builder/build.sh:125]: Let's see if the linting will pass with this:

```
buildpacks=(${buildpack_root}/*)
```

Comment (n0n0x) [reply rootfs/builder/build.sh:125]: according to this: https://www.shellcheck.net/ it won't pass:

image

but we can ignore this check by adding a comment:

image

Comment (Cryptophobia) [reply rootfs/builder/build.sh:125]: Okay, perfect @n0n0x ! Can you add the comment and rerun the CI and will merge as soon as it passes! 👍

Comment (n0n0x) [reply rootfs/builder/build.sh]: pushed!

Comment (Cryptophobia) [reply rootfs/builder/build.sh]: TY @n0n0x for this fix! Waiting for CI to pass.

Going to include this in the next release. It has been buggy for users.

Comment (Cryptophobia) [reply rootfs/builder/build.sh]: Hi @n0n0x , build failed. I think due to misspelling `spellcheck` instead of `shellcheck` of the comment. Can you please fix real quick when you can? :)

File: rootfs/builder/build.sh
Comment (Cryptophobia) [inline rootfs/builder/build.sh:125]: should be ```# shellcheck disable=SC2206```

Comment (n0n0x) [reply rootfs/builder/build.sh]: lol, sorry! pushed!

Comment (Cryptophobia) [reply rootfs/builder/build.sh]: Awesome! We have a GREEN build. :)

**Comment by kingdonb** [comment]
A report came through #support in slack today that build detection in the master branch is still not working, I have not been able to confirm this myself but I wanted to note it here. I added an item to stepsize so we don't lose track of it.

[stepsize] Sounds like build detection is still borked after slugbuilder#23 merged

**Comment by kingdonb** [comment]
#24 has the rest of the fix

**Pull request merged**
